New Common Controls Policy: Whether your users can add account recovery information (#611)
cisagov/ScubaGoggles: scubagoggles/baselines/commoncontrols.md

4.	Select the radio button option: **"Replace conflicting unmanaged accounts with managed ones."**
5.	Click **Save.**

## 8. Account Recovery Options

> This section covers self-recovery setting for users and super admins in the Google Admin console as well as user added account recovery information.

Reviewer: Wording is a bit awkward and doesn't add much. Maybe use this space to provide background on the feature for any users who may not be familiar?
```suggestion
This section addresses the Google Workspace account self-recovery feature. When enabled, this feature allows users to add a recovery email or phone number with which they can reset their own accounts if needed, without needing to contact a workspace admin for support.
```

### Policies

#### GWS.COMMONCONTROLS.8.1v0.4

> Account self-recovery for Super Admins SHALL be disabled.

Reviewer: Looks like super admins isn't capitalized in the docs or admin center. 
```suggestion
Account self-recovery for super admins SHALL be disabled.
```

  - [T1556: Modifying Authentication Process](https://attack.mitre.org/techniques/T1556/)
    - [T1556:006: Modifying Authentication Process: Multi-Factor Authentication](https://attack.mitre.org/techniques/T1556/006/)

#### GWS.COMMONCONTROLS.8.2v0.4

> Account self-recovery for User and Non-Super Admins SHALL be disabled.

Reviewer: Change capitalization to be grammatically correct and to match the admin center.
```suggestion
Account self-recovery for users and non-super admins SHALL be disabled.
```

- _Rationale:_ If enabled, a user could add a personal email or phone number for account recovery. Disabling this feature makes account recovery go through official channels, making it more difficult for a potential adversary to compromise an account.
- _Last modified:_ February 2025

- MITRE ATT&CK TTP Mapping
  - [T1556: Modifying Authentication Process](https://attack.mitre.org/techniques/T1556/)
    - [T1556:006: Modifying Authentication Process: Multi-Factor Authentication](https://attack.mitre.org/techniques/T1556/006/)

#### GWS.COMMONCONTROLS.8.3v0.4
Ability to add recovery information SHOULD be disabled.

> - _Rationale:_ If enabled, a user could add a personal email or phone number for account recovery. Disabling this feature prevents a user from adding PII to their organizational account, making it more difficult for a potential adversary to compromise an account.

Reviewer: I'm not sure this makes it more or less difficult to compromise the account. The only danger I see here is that if the user's GWS account is compromised, then that user's PII will be leaked as well.
```suggestion
- _Rationale:_ If enabled, a user could add a personal email or phone number for account recovery. Disabling this feature prevents a user from adding PII to their organizational account, making it more difficult for a potential adversary to steal PII in the event of a compromise.
```

- _Last modified:_ February 2025

- MITRE ATT&CK TTP Mapping
  - [T1556: Modifying Authentication Process](https://attack.mitre.org/techniques/T1556/)

>     - [T1556:006: Modifying Authentication Process: Multi-Factor Authentication](https://attack.mitre.org/techniques/T1556/006/)

Reviewer: Since this isn't a thing that enables account compromise and the risk is instead data leakage, https://attack.mitre.org/techniques/T1530/ might be more applicable.

Author: Agreed, updated the TTPs

7.  Click **Save**.
8.  Ask your Super Admins to set up a recovery phone number or email address for receiving password recovery instructions.

#### GWS.COMMONCONTROLS.8.2v0.4 Instructions
To disable User account self-recovery:

1.  Sign in to https://admin.google.com as an administrator.
2.  Select **Security** -\> **Authentication.**
3.  Select **Account Recovery**.
4.  Click **User account recovery**.

> 5.  To apply the setting to all your users, leave the top OU selected. Otherwise, select a child OU or a configuration group.
> 6.  Deselect the **Allow users and all non-super Admins to recover their account** checkbox.
> 7.  Click **Save**.

Reviewer: This control applies to all users, this step here implies that it doesn't (same goes for step 5 of GWS.COMMONCONTROLS.8.2 but it won't let me comment directly on that).
```suggestion
5.  Deselect the **Allow users and all non-super admins to recover their account** checkbox.
6.  Click **Save**.
```

#### GWS.COMMONCONTROLS.8.3v0.4 Instructions
1.  Sign in to https://admin.google.com as an administrator.
2.  Select **Security** -\> **Authentication.**
3.  Select **Account Recovery**. 
4.  Click **Recovery Information**.

> 4.  To apply the setting to all your users, leave the top OU selected. Otherwise, select a child OU or a configuration group.
> 5.  Deselect the **Allow admins and users to add recovery email information to their account** checkbox.
> 6.  Deselect the **Allow admins and users to add recovery phone information to their account** checkbox.
> 7.  Click **Save**.

Reviewer: This control applies to all users, this step here implies that it doesn't.
```suggestion
5.  Deselect the **Allow admins and users to add recovery email information to their account** checkbox.
6.  Deselect the **Allow admins and users to add recovery phone information to their account** checkbox.
7.  Click **Save**.
```
